test/Project.toml

```diff
@@ -1,6 +1,8 @@
 [deps]
 Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
 CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
+Convex = "f65535da-76fb-5f13-bab9-19810c17039a"
+DelimitedFiles = "8bb1440f-4735-579b-a4ab-409b98df4dab"
```

Comment:
I guess we need ECOS also here: https://github.com/trixi-framework/Trixi.jl/actions/runs/8950168743/job/24585250457?pr=1908#step:7:2058

Reply:
I've made adjustments for both errors. Could you let the test run again so that I can see if this is resolved?

Reply:
The tests run automatically again after pushing a commit. You can see the status of the tests at the bottom of the PR.

Reply:
I see. It seems that the tests are not triggered automatically when the commits are made by me. Here is the status at the bottom of the PR from my view: `7 workflows awaiting approval`

Reply:
Ah, yes. This is because it's your first PR to Trixi.jl. Unfortunately, I also don't have the rights to approve the workflows.

Reply:
@warisa-r You can just adapt the test values to what you now get, this is the usual workflow.

Reply:
I recently have. The same failure messages still showed up 😬

```
Test Failed at /home/runner/work/Trixi.jl/Trixi.jl/test/test_trixi.jl:83
  Expression: isapprox(l2_expected, l2_actual, atol = 1.1102230246251565e-13, rtol = 1.4901161193847656e-8)
   Evaluated: isapprox(0.014139243437868486, 0.014139244532882265; atol = 1.1102230246251565e-13, rtol = 1.4901161193847656e-8)
```

Reply:
Hm strange, this seems to be hardware dependent. You can just use the values from CI.

Reply:
Seems to be the same case for the error above too. As soon as I see the value of` a_matrix` from CI, of `test_unit.jl`, I'll update the file accordingly.